docs/tutorials/vae.rst

```diff
@@ -127,50 +124,38 @@ another computation graph from scratch and reuse the Variables (weights here).
 If there are many stochastic nodes in the model, this process will be really
 painful.
 
-**ZhuSuan has a novel solution for this.** To observe any stochastic nodes,
-pass a dictionary mapping of ``(name, Tensor)`` pairs when constructing
-:class:`~zhusuan.model.base.BayesianNet`. This will assign observed values
-to corresponding ``StochasticTensor`` s. For example, to observe
-a batch of images ``x_batch``, write::
+**ZhuSuan has a novel solution for this.** To observe any stochastic nodes, 
```

Comment:
following the old version, we can write (make sure the internal reference link works) "However, we usually need to pass different observations to the same `BayesianNet` more than once. To achieve this, ZhuSuan provides a new class called `MetaBayesianNet` to represent the meta version of `BayesianNet` which can repeatly produce `BayesianNet` objects by accepting different observations. The recommended way to construct a `MetaBayesianNet` is by the ... decorator. 

and refer them back to the basic concepts tutorial for more information (though it haven't been updated, let's do that later)